src/main/java/net/commments/object/CommentTable.java

```diff
@@ -0,0 +1,5 @@
+package net.commments.object;
+
+public interface CommentTable {
```

Comment:
Why did you create this interface? It's not used in the test. Please remove.

Reply:
it will be used in my next tests

Reply:
@lvrub then add in the next test and remove it now.

Reply:
done